documentation/tutorials/get-started-with-spark.md

```diff
@@ -60,7 +60,9 @@ uses under the hood to allow users to write in the DSL syntax described above.
 ```elixir
 defmodule MyLibrary.Validator.Dsl do
   defmodule Field do
-    defstruct [:name, :type, :transform, :check]
+    # The __spark_metadata__ field is required for Spark entities
```

Comment:
its not required though is it? Its optional, but will add extra information if present.